usmqe/api/tendrlapi/cephapi.py

```diff
@@ -19,3 +19,314 @@ def import_cluster(self, nodes, asserts_in=None):
             asserts_in (dict): assert values for this call and this method
         """
         super().import_cluster(nodes, "ceph", asserts_in)
+
+    def create_pool(self,
+                    cluster,
+                    name,
+                    pg_num,
+                    min_size,
+                    size,
+                    pool_type=None,
+                    erasure_code_profile=None,
+                    quota_enabled=None,
+                    quota_max_objects=None,
+                    quota_max_bytes=None):
+        """ Create Ceph pool.
+
+        Name:        "create_pool",
+        Method:      "POST",
+        Pattern:     ":cluster_id:/CephCreatePool",
+
+        Args:
+            cluster: Cluster ID (String)
+            name: Pool name. (String)
+            pg_num: Number of placement groups. (Int)
+            min_size: Minimum number of replicas required for I/O
+                      in degraded state (Int)
+            size: Minimum number of replicas required for I/O. (Int)
+            pool_type: Type of the Ceph pool(ec or replicated) (String)
+            erasure_code_profile: For erasure pools only.It must be
+                                  an existing profile. (String)
+            quota_enabled: Enable quota for the pool. (Bool)
+            quota_max_objects: Maximum number of object in pool. (Int)
+            quota_max_bytes: Maximum number of bytes in pool. (Int)
+        """
+        pattern = "{}/CephCreatePool".format(cluster)
+        pool_data = {
+            "Pool.poolname": name,
+            "Pool.pg_num": pg_num,
+            "Pool.min_size": min_size,
+            "Pool.size": size,
+            }
+        if pool_type:
+            pool_data["Pool.type"] = pool_type
+        if erasure_code_profile:
```

Comment:
That depends if tendrl api check that. I am pretty sure that if the pool type is not 'ec' erasure code profile will be ignored.
That said there will be no difference between valid non-ec settings with or without erasure code profile set.